parse_date.re

```diff
@@ -2295,7 +2297,7 @@ timelib_time *timelib_parse_from_format_with_map(const char *format, const char
 						add_pbf_error(s, TIMELIB_ERR_NO_TWO_DIGIT_YEAR, "A two digit year could not be found", string, begin);
 						break;
 					}
-					
+
```

Comment:
Please don't mix functional changes with white space changes in the same commit. Feel free to separate them out to a different commit in the same PR.